lib/hexpm/web/templates/package/show.html.eex

```diff
@@ -38,9 +38,11 @@ tools = Keyword.take(tools, compatible_tools)
 </h2>
 
 <%= if retirement = @current_release && @current_release.retirement do %>
-  <div class="alert alert-danger" style="margin-top: 40px" role="alert">
-    <%= retirement_html(retirement) %>
-  </div>
+  <%= unless retirement_message_empty?(retirement) do %>
+    <div class="alert alert-danger" style="margin-top: 40px" role="alert">
+      <%= retirement_html(retirement) %>
+    </div>
+  <% end %>
```

Comment:
We should still always show an message but we need to improve its formatting in some cases of course.

Reply:
Sure. How about "This package has been retired." ?

Reply:
Maybe something like this: "Retired package, reason: #{reason} - #{message}"?

Reply:
Sure. In the case of "other" with no message, do you think it should read, `Retired Package, reason: other` or just `Retired Package` ?
It seems the formatter is making "Other" nil in many cases.

Reply:
Just "Retired package" is probably better in that case 👍

Reply:
Sounds good. I'll push those changes up later today.